dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -1825,6 +1831,31 @@ sub fetch_para {
     )
     {
       last if $self->{lastline} =~ /^\S/ && @{ $self->{line} } && $self->{line}->[-1] eq "";
+      if ($self->{lastline}=~/^#[ \t]*(if|ifn?def|elif|else|endif)\b/) {
```

Comment:
The upcoming C++23 and C23 standards also add `#elifdef`, `#elifndef`, though I'm not sure you need any of the `#else` variants here.

https://en.cppreference.com/w/cpp/preprocessor
https://en.cppreference.com/w/c/preprocessor

Reply:
I can add support for them, but as you stated the only thing they would be used for would be to detect one without a corresponding starting 'if'.

Reply:
Added support for it.